metrics.go

```diff
@@ -64,23 +63,34 @@ var metrics = struct {
 		metric.WithExplicitBucketBoundaries(0.001, 0.003, 0.005, 0.01, 0.03, 0.05, 0.1, 0.3, 0.5, 1.0, 2.0, 3.0, 4.0, 5.0),
 		metric.WithUnit("s"),
 	)),
-	validationDuplicateMessages:       measurements.Must(meter.Int64Counter("f3_validation_duplicate_messages", metric.WithDescription("Number of duplicate GPBFT messages validated."))),
-	validationDuplicateJustifications: measurements.Must(meter.Int64Counter("f3_validation_duplicate_justifications", metric.WithDescription("Number of duplicate GPBFT justifications validated."))),
+	validatedMessages: measurements.Must(meter.Int64Counter("f3_validated_messages",
+		metric.WithDescription("Number of validated GPBFT messages."))),
 }
 
-func recordValidationDuplicates(ctx context.Context, msg gpbft.ValidatedMessage) {
+func recordValidatedMessage(ctx context.Context, msg gpbft.ValidatedMessage) {
 	// The given msg and its validated value should never be nil; but defensively
 	// check anyway.
 	if msg == nil || msg.Message() == nil {
 		return
 	}
+
+	// Count once for total number of validated messages.
+	metrics.validatedMessages.Add(ctx, 1)
```

Comment:
remove this?

Reply:
Would we get total validated messages without it?

Reply:
Yes? We update this at the end and can just sum disregarding labels.

Reply:
Sum alone would return sum of duplicates i would imagine. I'll test this tomorrow to be sure this line is needed. Thanks

Reply:
We can also sum on `duplicate_message`, etc. That would give us both the count of unique messages and the count of duplicates (separately). I'm flagging this because we're currently counting all messages twice, which is definitely wrong.

Reply:
Thank you for your patience; I see now.